meshtastic/mesh_interface.py

```diff
@@ -129,7 +129,6 @@ def showInfo(self, file=sys.stdout):  # pylint: disable=W0613
 
                 # use id as dictionary key for correct json format in list of nodes
                 nodeid = n2["user"]["id"]
-                n2["user"].pop("id")
```

Comment:
I'm not sure why we did this other than to clean up the display, but it meant if this got called and later something else needed the ID, it would fail.